### ISSUE TITLE
[wordpress-bitnami-aws-lightsail] Services are shutting down automatically again and again

### ISSUE THREAD

**Issue by dangerdon78** [bitnami/vms#1192]
### Platform

AWS

### bndiagnostic ID [know more about bndiagnostic ID](https://docs.bitnami.com/general/how-to/understand-bndiagnostic/)

8c3e01c9-1a89-8de8-cd8c-29a62d82eb38

### bndiagnostic output

    ✓ Resources: No issues found
    ? Connectivity: Found possible issues
    ✓ Mariadb: No issues found
    ? Processes: Found possible issues
    ? Wordpress: Found possible issues
    ? Apache: Found possible issues
    ✓ Php: No issues found
 

 
[Connectivity]

Server ports 22, 80 and/or 443 are not publicly accessible. Please check the 
following guide to open server ports for remote access:

https://docs.bitnami.com/general/faq/administration/use-firewall/

[Processes]

Press [Enter] to continue:
One or more component's processes are not running:
```
apache not running mariadb not running php-fpm not running
```
You can try to restart the process with the following command:
 
 $ sudo /opt/bitnami/ctlscript.sh start COMPONENT_NAME

[Wordpress]

Found recent WordPress plugin related error messages in the Apache error log.
```
[Thu Oct 19 05:39:51.917870 2023] [autoindex:error] [pid 1019825:tid 
140676150785792] [client **ip_address**:36900] AH01276: Cannot serve directory 
/opt/bitnami/wordpress/wp-includes/js/tinymce/plugins/compat3x/css/: No matching 
DirectoryIndex (index.html,index.html,index.htm,index.php) found, and 
server-generated directory index forbidden by Options directive, referer: 
/wp-includes/js/tinymce/plugins/compat3x/css/index.php

```
Please check the following guide to deactivate plugins:
 
https://developer.wordpress.org/cli/commands/plugin/deactivate/
Press [Enter] to continue:
[Apache]

Found recent error or warning messages in the Apache error log.
```
[Thu Oct 19 10:42:13.614573 2023] [authz_core:error] [pid 1021293:tid 
140674884044544] [client **ip_address**:57178] AH01630: client denied by server 
configuration: /opt/bitnami/wordpress/.user.ini
 [Thu Oct 19 10:50:58.887596 2023] [proxy_fcgi:error] [pid 1021293:tid 
140675974670080] [client **ip_address**:52492] AH01067: Failed to read FastCGI 
header
 [Thu Oct 19 10:50:58.888712 2023] [proxy_fcgi:error] [pid 1021293:tid 
140675974670080] (104)Connection reset by peer: [client **ip_address**:52492] 
AH01075: Error dispatching request to : 

```
Please check the following guide to troubleshoot server issues:
 
https://docs.bitnami.com/general/apps/wordpress/troubleshooting/debug-erro
rs-apache/
 

===== End of bndiagnostic tool output =====

### bndiagnostic was not useful. Could you please tell us why?

My server, db, and php-fpm are shutting down automatically

### Describe your issue as much as you can

My services are shutting down automatically again and again.
**_apache not running mariadb not running php-fpm not running._**\

When i restart using this command: sudo /opt/bitnami/ctlscript.sh restart, the services starts normally. But after 3-5 days the services are shutting down automatically again and again.

May i know the problem?

Is there any security vulnerability?

**Comment by gongomgra** [comment]
Hi @dangerdon78,

Thanks for using Bitnami. Ligthsail instances are known to suffer performance issues due to the kind of those instances ([burstable instances](https://lightsail.aws.amazon.com/ls/docs/en_us/articles/amazon-lightsail-frequently-asked-questions-faq#amazon-lightsail-faq-instances)). We recommend you to check this in a more specialized forum or with the AWS support team directly.

Apart from the above, it looks like the bndiagnostic tool found some issues on your installation. Did you have any issue trying to fix them?

**Comment by dangerdon78** [comment]
My Instance is running in sustainable zone and not in burstable zone. 
"you to check this in a more specialized forum or with the AWS support team directly" _ This is not an answer or solution to my problem. Bitnami is managing this setup on AWS. It is also Bitnami's responsibility to solve problems with installation or server issues.

"Apart from the above, it looks like the bndiagnostic tool found some issues on your installation." - Can you please tell which installation has issues? Is it WordPress, Plugin, SSL or the Instance?

**Comment by gongomgra** [comment]
Hi @dangerdon78,

Although we publish our images in AWS, we don't have any access or control privileges in the servers you run using our images as base images. Because of that, there are some topics in which we can't be of much help, like those relative to performance, where either the instance type or your customization of the base image maybe causing these errors.

According to the bndiagnostic output, there are errors related to WordPress plugins. I also found the next output in the support bundle: did you modify anything in the configuration of gonit so it can't monitor the expected services?

```
-----------------------------------
Check gonit status
-----------------------------------
Running: gonit status
In: /opt/bitnami

Output:


Uptime                         1351h27m20s
Last Check                     2023-10-19 11:06:15.363990986 +0000 UTC m=+4865207.802362105
Next Check                     2023-10-19 11:08:15.363990986 +0000 UTC m=+4865327.802362105
Pid                            2298
Pid File                       /var/run/gonit.pid
Control File                   /etc/gonit/gonitrc
Socket File                    /var/run/gonit.sock
Log File                       /var/log/gonit.log
Process 'apache'
  monitoring status                        Not monitored

Process 'mariadb'
  monitoring status                        Not monitored

Process 'php-fpm'
  monitoring status                        Not monitored
``` 

Apart from the above, I visited your website and it seems to be working properly now. Did you manage to fix your issue?

**Comment by iba-1** [comment]
Multiple issues have been opened regarding Lightsail instances going down randomly and the answer provided from Bitnami was always the same:
https://github.com/bitnami/vms/issues/39
https://github.com/bitnami/vms/issues/89
https://github.com/bitnami/vms/issues/136
https://github.com/bitnami/vms/issues/167
https://github.com/bitnami/vms/issues/183
https://github.com/bitnami/vms/issues/902
https://github.com/bitnami/vms/issues/907
https://github.com/bitnami/vms/issues/1127
https://github.com/bitnami/vms/issues/1136

It seemed a bit too strange that this was happening out of the blue, and to this many users, only on server started using Bitnami Wordpress Image, so we dug deeper.
Could this be related to this bug: https://bugs.php.net/bug.php?id=77023, which appear to be fixed on php 8.0?

**Comment by gongomgra** [comment]
Hi @iba-1,

Thanks for your message. It can be related, but I'm afraid we don't have control on the AWS Lightsail release cadence, and those images may still be using PHP 7.x. This can also be related with the issue linked below

https://github.com/bitnami/vms/issues/927
